### PR TITLE
Add readme and notes to the chart

### PR DESCRIPTION
**Pull request by kfox1111**
This adds a basic readme file and notes to the helm chart

### CODE REVIEW

File: deploy/rbd/helm/templates/NOTES.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-The Ceph RBD Container Storage Interface has been deployed.
+Examples on how to configure a storage class and start using the driver are here:
+https://github.com/ceph/ceph-csi/tree/master/examples/rbd
```
Comment (rootfs) [inline deploy/rbd/helm/templates/NOTES.txt]: match branch name

File: deploy/rbd/helm/Chart.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ appVersion: "1.0.0"
 description: "Container Storage Interface (CSI) driver,
 provisioner, and attacher for Ceph RBD"
 name: ceph-csi-rbd
-version: 0.3.0
+version: 0.4.0
```
Comment (rootfs) [inline deploy/rbd/helm/Chart.yaml]: github doesn't allow me to comment below, but can you also update the branch name

File: deploy/rbd/helm/Chart.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ appVersion: "1.0.0"
 description: "Container Storage Interface (CSI) driver,
 provisioner, and attacher for Ceph RBD"
```
Comment (rootfs) [inline deploy/rbd/helm/Chart.yaml]: snapshotter too

File: deploy/rbd/helm/templates/NOTES.txt (hunk 1 above)
Comment (kfox1111) [reply deploy/rbd/helm/templates/NOTES.txt]: oh, right.

**Comment by rootfs** [comment]
also failed on linter:
```
./deploy/rbd/helm/README.md:5: MD012 Multiple consecutive blank lines
```

**Pull request merged**